.github/docs/entrypoint.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+set -u
+
+SSH_PATH="/root/.ssh"
+
+mkdir "$SSH_PATH"
+touch "$SSH_PATH/known_hosts"
+
+echo "$DEPLOY_SSH_KEY" > "$SSH_PATH/id_rsa"
+
+chmod 700 "$SSH_PATH"
+chmod 600 "$SSH_PATH/known_hosts"
+chmod 600 "$SSH_PATH/id_rsa"
+
+eval $(ssh-agent)
+ssh-add "$SSH_PATH/id_rsa"
+
+ssh-keyscan -t rsa "github.com" >> "$SSH_PATH/known_hosts"
+echo "StrictHostKeyChecking no" >> "$SSH_PATH/config"
+
+# Change into the build directory
+cd $BUILD_DIR
+
+git config --global user.name "jonwinton" && \
+git config --global user.email "jonwinton@users.noreply.github.com" && \
+npm install && \
+npm run build && \
+npx gh-pages -d build/clay-cli --repo "git@github.com:$GITHUB_REPOSITORY.git"
```

Comment:
A question here. Do we need to delete this line on the `package.json`?
https://github.com/clay/claycli/blob/master/website/package.json#L10
Since we gonna do the deploy with the action.

Reply:
I wouldn't because that still works locally. This is a special case for when the action is running.